src/Compilers/Core/Portable/Symbols/INamedTypeSymbol.cs

```diff
@@ -167,7 +167,7 @@ public interface INamedTypeSymbol : ITypeSymbol
         bool MightContainExtensionMethods { get; }
 
         /// <summary>
-        /// If this is a tuple type symbol, returns the symbol for its underlying type (ie. without element names).
+        /// If this is a tuple type with element names, returns the symbol for the tuple type without names.
```

Comment:
>If this is a tuple type with element names, returns the symbol for the tuple type without names. [](start = 12, length = 96)

Is this accurate for long tuples?

Reply:
Yes, that should be correct for long tuples as well.

---
In reply to: [382318717](https://github.com/dotnet/roslyn/pull/41828#discussion_r382318717) [](ancestors = 382318717)